basics/Main.java

```diff
@@ -0,0 +1,80 @@
+// random number generator
+import java.util.*; 
+//time and formating
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.Month;
+import java.time.format.*;
+
+public class Main {
+  public static void main(String[] args){
+    //test pluralize
+    int dogCount = 1;
+    System.out.println("I own " + dogCount + " " + pluralize("dog", dogCount) + ".");
+    int catCount = 2;
+    System.out.println("I own " + catCount + " " + pluralize("cat", catCount) + ".");
+    int turtleCount = 0;
+    System.out.println("I own " + turtleCount + " " + pluralize("turtle", turtleCount) + ".");
+
```

Comment:
this wasn't mentioned so no points off, but generally when you are "testing" you code, you want to try it multiple times. try calling this method a few times with different numbers to make sure it works every time